ui/plugin-admin/src/components/PluginDetail.js

```diff
@@ -50,7 +52,19 @@ const PluginDetail = () => {
       opened={!!showDetailsFor}
       onClose={onPanelClose}
       size="large"
-      heading={plugin?.disabled ? plugin?.name + " DISABLED" : plugin?.name}
+      heading={
+        plugin?.disabled ? (
```

Comment:
Why the special handling for disabled plugins here? The `PluginConditionIcon` component takes care of this, doesn't it?